src/main/java/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.java

```diff
@@ -95,7 +95,7 @@
     protected ChunkingFilter chunkingFilter;
     protected MemberCachePolicy memberCachePolicy = MemberCachePolicy.ALL;
 
-    private DefaultShardManagerBuilder(@Nullable String token, int intents)
+    protected DefaultShardManagerBuilder(@Nullable String token, int intents)
```

Comment:
I would prefer to add a second constructor which accepts `EnumSet<GatewayIntent>` instead of an int. This constructor can be used to set invalid intents which should not be allowed even by subclasses.

Reply:
If you intend to let classes extend this, when methods like `applyDefault` and `applyLight` should also be changed to protected.